### PR TITLE
Refactor/storybook use infinite scroll

### LINKED ISSUE #2506: Deprecated props are being used in the Storybook `use-infinite-scroll` hook's page.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.4.8

### Description

Deprecated props are being used in the Storybook `use-infinite-scroll` page.
Use `fontSize` instead of `size`.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by ChinoUkaegbu**
Closes #2506

## Description

Replaced size with fontSize in Storybook use-infinite-scroll hook and updated the docs

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by Shunii85** [comment]
Hi @ChinoUkaegbu , thanks for your PR!!

I'm sorry but you don't have to change `size` prop used in the ` Heading` component.

The `size` prop used in the `Icon` component is deprecated.
https://github.com/yamada-ui/yamada-ui/blob/main/packages/components/icon/src/icon.tsx#L21

**Comment by ChinoUkaegbu** [comment]
I see. So the Heading 'size' should remain the same but the Loading 'size' should be changed to 'fontSize'?

File: docs/contents/hooks/use-infinite-scroll.ja.mdx [hunk 1]
```diff
@@ -32,7 +32,7 @@ return (
       .map((_, index) => (
         <Card key={index}>
           <CardHeader>
-            <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+            <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.ja.mdx:35]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.ja.mdx [hunk 2]
```diff
@@ -95,7 +95,7 @@ return (
         .map((_, index) => (
           <Card key={index}>
             <CardHeader>
-              <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+              <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.ja.mdx:98]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.ja.mdx [hunk 3]
```diff
@@ -146,7 +146,7 @@ return (
       .map((_, index) => (
         <Card key={index}>
           <CardHeader>
-            <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+            <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.ja.mdx:149]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.ja.mdx [hunk 4]
```diff
@@ -194,7 +194,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.ja.mdx:197]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.ja.mdx [hunk 5]
```diff
@@ -245,7 +245,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.ja.mdx:248]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 6]
```diff
@@ -32,7 +32,7 @@ return (
       .map((_, index) => (
         <Card key={index}>
           <CardHeader>
-            <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+            <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:35]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 7]
```diff
@@ -95,7 +95,7 @@ return (
         .map((_, index) => (
           <Card key={index}>
             <CardHeader>
-              <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+              <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:98]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 8]
```diff
@@ -146,7 +146,7 @@ return (
       .map((_, index) => (
         <Card key={index}>
           <CardHeader>
-            <Heading size="md">俺を誰だと思っていやがるッ!!</Heading>
+            <Heading fontSize="md">俺を誰だと思っていやがるッ!!</Heading>
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:149]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 9]
```diff
@@ -194,7 +194,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:197]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 10]
```diff
@@ -245,7 +245,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:248]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 11]
```diff
@@ -293,7 +293,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:296]: Do not change this line.

File: docs/contents/hooks/use-infinite-scroll.mdx [hunk 12]
```diff
@@ -349,7 +349,7 @@ return (
...
```
Comment (hirotomoyamada) [inline docs/contents/hooks/use-infinite-scroll.mdx:352]: Do not change this line.

File: stories/hooks/use-infinite-scroll.stories.tsx [hunk 13]
```diff
@@ -38,7 +38,7 @@ export const basic = () => {
         .map((_, index) => (
           <Card key={index}>
             <CardHeader>
-              <Heading size="md">『ドラゴンボール』（DRAGON BALL）</Heading>
+              <Heading fontSize="md">『ドラゴンボール』（DRAGON BALL）</Heading>
```
Comment (hirotomoyamada) [inline stories/hooks/use-infinite-scroll.stories.tsx:41]: Do not change this line.

File: stories/hooks/use-infinite-scroll.stories.tsx [hunk 14]
```diff
@@ -90,7 +90,7 @@ export const withRoot = () => {
           .map((_, index) => (
             <Card key={index}>
               <CardHeader>
-                <Heading size="md">『ドラゴンボール』（DRAGON BALL）</Heading>
+                <Heading fontSize="md">『ドラゴンボール』（DRAGON BALL）</Heading>
```
Comment (hirotomoyamada) [inline stories/hooks/use-infinite-scroll.stories.tsx:93]: Do not change this line.

File: stories/hooks/use-infinite-scroll.stories.tsx [hunk 15]
```diff
@@ -134,7 +134,7 @@ export const withRootMargin = () => {
         .map((_, index) => (
           <Card key={index}>
             <CardHeader>
-              <Heading size="md">『ドラゴンボール』（DRAGON BALL）</Heading>
+              <Heading fontSize="md">『ドラゴンボール』（DRAGON BALL）</Heading>
```
Comment (hirotomoyamada) [inline stories/hooks/use-infinite-scroll.stories.tsx:137]: Do not change this line.

File: stories/hooks/use-infinite-scroll.stories.tsx [hunk 16]
```diff
@@ -175,7 +175,7 @@ export const withThreshold = () => {
...
```
Comment (hirotomoyamada) [inline stories/hooks/use-infinite-scroll.stories.tsx:178]: Do not change this line.

File: stories/hooks/use-infinite-scroll.stories.tsx [hunk 17]
```diff
@@ -216,7 +216,7 @@ export const withInitialLoad = () => {
...
```
Comment (hirotomoyamada) [inline stories/hooks/use-infinite-scroll.stories.tsx:219]: Do not change this line.

**Comment by ChinoUkaegbu** [comment]
I've made the changes from the review!

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**